app/covidbed/model/resource.py

```diff
@@ -0,0 +1,33 @@
+from .abc import db, Base
+from .orga import Organization, Platform
+
+
+class Contact(Base):
+    __tablename__ = 'ress_contact'
+    id = db.Column(db.Integer, primary_key=True)
+    firstname = db.Column(db.String(30), nullable=False)
+    lastname = db.Column(db.String(30), nullable=False)
+    email = db.Column(db.String(30), nullable=False)
+    phone_number = db.Column(db.String(10), nullable=False)
+    comment = db.Column(db.Text, nullable=True)
+
+
+class Availability(Base):
+    __tablename__ = 'ress_availability'
+
+    id = db.Column(db.Integer, primary_key=True)
+    date = db.Column(db.DateTime, nullable=False)
+    functional_unit = db.Column(db.String(25), nullable=True)
```

Comment:
Is the functional unit the entity owning the resource (just to clarify)?

Reply:
May be the functional_unit is the service of the organization. (rea, urgence, ...). It's optional,
because we don't know the granyularity used by the project.

Reply:
Ok, clear.